### ISSUE TITLE
Invalid content type 'text/html' in response after calling analyzer endpoint

### ISSUE THREAD

**Issue by v1nc3n4** [microsoft/presidio#621]
Hi,

**Describe the bug**
I'm using the Docker image of the analyzer `mcr.microsoft.com/presidio-analyzer:latest`.
When analyzing a text with the REST API on default port 3000, the HTTP response has an unexpected content type `text/html`, which leads to failure when decoding the payload with libraries such as Spring WebClient.

**To Reproduce**

1. Send request

```
POST /analyze HTTP/1.1
Content-Type: application/json
User-Agent: PostmanRuntime/7.26.10
Accept: */*
Host: localhost:3000
Accept-Encoding: gzip, deflate, br
Connection: keep-alive
Content-Length: 72

{
  "text": "hello world, my name is Jane Doe.",
  "language": "en"
}
```

2. Response received

```
Content-Type: text/html; charset=utf-8
Content-Length: 148
Server: Werkzeug/1.0.1 Python/3.8.8
Date: Thu, 18 Mar 2021 15:00:51 GMT

[
{
"analysis_explanation": null,
"end": 32,
"entity_type": "PERSON",
"score": 0.85,
"start": 24
}
]
```

**Expected behaviour**
The response has a `Content-Type: application/json` header instead of a `Content-Type: text/html; charset=utf-8` header.

What am I doing wrong?
Thanks in advance for your help!
BR

**Comment by navalev** [comment]
thanks you @v1nc3n4. issue is logged and we will follow up on it

**Comment by navalev** [comment]
@v1nc3n4 issue is resolved in #626

**Comment by v1nc3n4** [comment]
Hi,
Thanks for your reactive support!
Best regards
